tests/Resources/Page/MarketplaceConsole/MarketplaceConsole.robot

```diff
@@ -0,0 +1,67 @@
+*** Settings ***
+Library         SeleniumLibrary
+Library         Collections
+
+*** Keywords ***
+Is SSO Login Page Visible
```

Comment:
we already have a common keyword like that. Could you check if that one is compatible here? (HCCLogin.robot)

Reply:
okay. No I think it's fine, I was just wondering if we could re-use the already existing one.